mockito-core/build.gradle

```diff
@@ -1,20 +1,15 @@
-
 plugins {
-    id 'base'
-    id 'biz.aQute.bnd.builder' version '7.0.0'
-    id 'ru.vyarus.animalsniffer' version '1.7.1'
+    id("mockito.library-conventions")
+    id("mockito.java-backward-compatibility-checks-conventions")
+    id("mockito.javadoc-conventions")
 }
 
 description = 'Mockito mock objects library core API and implementation'
 base.archivesName = 'mockito-core'
 
-
-apply from: "$rootDir/gradle/java-library.gradle"
 apply from: "$rootDir/gradle/coverage.gradle"
```

Comment:
Do we want to tackle all `apply from` applications in this PR? I don't mind doing them in a separate PR as well, but curious to what your preferred ordering of work is.

Reply:
Indeed, I prefer to proceed with the rest in another PR. I stopped where the changes makes sense.

Especially for coverage, where I noticed we may be able to [simply the aggregation](https://github.com/orgs/mockito/projects/1/views/1?pane=issue&itemId=80276438).